packages/api/src/schema/api-schema.yaml

```diff
@@ -1033,9 +1033,53 @@ components:
       properties:
         name:
           type: string
+          description: Display name
+          example: name
         canPublish:
           type: boolean
           description: Whether a user is allowed to publish audio/video tracks
+          example: true
+    create-room-response:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the room
+          example: d32ae9e6-c459-4931-9898-e86e2f5e7e16
+    room-user-response:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the user
+          example: d32ae9e6-c459-4931-9898-e86e2f5e7e16
```

Comment:
Are user IDs also UUIDs? Thought they could be sth else

Reply:
Yep, I just generate them here: https://github.com/livepeer/studio/blob/master/packages/api/src/controllers/room.ts#L201